sdks/python/setup.py

```diff
@@ -442,7 +442,7 @@ def get_portability_package_data():
               # Until this issue is closed
               # https://github.com/googleapis/google-cloud-python/issues/10566
               'google-auth>=1.18.0,<3',
-              'google-auth-httplib2>=0.1.0,<0.2.0',
```

Comment:
we should not use <= in upper bounds. https://cwiki.apache.org/confluence/display/BEAM/Dependency+management+guidelines+for+Beam+Python+SDK+maintainers

Reply:
should we do `<0.3.0` then?

Reply:
yes, that would be better; we'd have to regenerate the requirements as well.

Reply:
Done.